<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NFMDemodGUI</class>
 <widget class="RollupWidget" name="NFMDemodGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NFM Demodulator</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>35</y>
     <width>242</width>
     <height>111</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="3" column="1">
     <widget class="QSlider" name="volume">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="rfBW">
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="deltaMinus">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Minus</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="squelchLabel">
      <property name="text">
       <string>Squelch</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="afLabel">
      <property name="text">
       <string>AF Bandwidth</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="rfBWLabel">
      <property name="text">
       <string>RF Bandwidth</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="squelch">
      <property name="minimum">
       <number>-60</number>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>-40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="rfBWText">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>12.5 k</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="afBWText">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>3 k</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="volumeLabel">
      <property name="text">
       <string>Volume</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="afBW">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="squelchText">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>-40 dB</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="volumeText">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>2.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="deltaUnits">
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ValueDial" name="deltaFrequency" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>SizeVerCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTip">
       <string>Demod shift frequency from center in Hz</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="ctcssLabel">
      <property name="text">
       <string>CTCSS</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="ctcss">
      <property name="toolTip">
       <string>Set CTCSS</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="ctcssText">
      <property name="toolTip">
       <string>CTCSS detected</string>
      </property>
      <property name="text">
       <string>--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
